### DISCUSSION TITLE
Need help on extraction of audit event logs and session command records

### DISCUSSION THREAD

**Discussion by little-plant** [gravitational/teleport#41940] [Q&A]
I'm working on automating the extraction of audit event logs and session command records in Teleport v6.2.3. I've made progress but need help streamlining the process. Any suggestions on efficient approaches? Thanks!

**Comment by benarent** [comment]
Teleport 6 is pretty old and is end-of-life, and we would recommend upgrading your cluster to the latest version. Following our Upgrade https://goteleport.com/docs/upgrading/

Saying that, most of the foundational ways to view / obtain audit logs haven't changed but might depend on your backend. https://goteleport.com/docs/connect-your-client/tsh/#listing-recordings and then mounting your audit backend to get the audit log data https://goteleport.com/docs/reference/audit/#events
